<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>APersoPiscine</class>
 <widget class="QWidget" name="APersoPiscine">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1900</width>
    <height>1000</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="Background">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1900</width>
     <height>1000</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../../Desktop/PeroPiscine/PersoPiscine Abonne.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="Cours">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>370</y>
     <width>191</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="Salaire">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>470</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="Stock">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>560</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="Back">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>910</y>
     <width>51</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QTabWidget" name="CoursTab">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>260</y>
     <width>1260</width>
     <height>630</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="tabShape">
    <enum>QTabWidget::Triangular</enum>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <property name="iconSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
   <widget class="QWidget" name="Ajout">
    <attribute name="title">
     <string>Afficher Abonne</string>
    </attribute>
    <widget class="QTableWidget" name="Affichera">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>190</y>
       <width>1141</width>
       <height>341</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Source Sans Pro</family>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="autoScroll">
      <bool>false</bool>
     </property>
     <property name="autoScrollMargin">
      <number>20</number>
     </property>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>160</number>
     </attribute>
     <column>
      <property name="text">
       <string>ID</string>
      </property>
      <property name="font">
       <font>
        <family>Source Sans Pro</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="background">
       <color alpha="0">
        <red>0</red>
        <green>0</green>
        <blue>255</blue>
       </color>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Nom</string>
      </property>
      <property name="font">
       <font>
        <family>Source Sans Pro</family>
        <pointsize>14</pointsize>
       </font>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Prenom</string>
      </property>
      <property name="font">
       <font>
        <family>Source Sans Pro</family>
        <pointsize>14</pointsize>
       </font>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Num Telephone</string>
      </property>
      <property name="font">
       <font>
        <family>Source Sans Pro</family>
        <pointsize>14</pointsize>
       </font>
      </property>
     </column>
     <column>
      <property name="text">
       <string>DDNaissance</string>
      </property>
      <property name="font">
       <font>
        <family>Source Sans Pro</family>
        <pointsize>14</pointsize>
       </font>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Adresse</string>
      </property>
      <property name="font">
       <font>
        <family>Source Sans Pro</family>
        <pointsize>14</pointsize>
       </font>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Email</string>
      </property>
      <property name="font">
       <font>
        <family>Source Sans Pro</family>
        <pointsize>14</pointsize>
       </font>
      </property>
     </column>
    </widget>
    <widget class="QComboBox" name="comboBox">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>50</y>
       <width>141</width>
       <height>41</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>ID</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Nom</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Prenom</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Numero Tel</string>
      </property>
     </item>
    </widget>
    <widget class="QComboBox" name="comboBox_2">
     <property name="geometry">
      <rect>
       <x>850</x>
       <y>50</y>
       <width>141</width>
       <height>41</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>ID</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Nom</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Prenom</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Numero Tel</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Date de naissance</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="IDC_3">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>60</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Rechercher :</string>
     </property>
    </widget>
    <widget class="QLabel" name="IDC_4">
     <property name="geometry">
      <rect>
       <x>800</x>
       <y>60</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Tri :</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>50</y>
       <width>170</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>../../Desktop/ADMIN/Recherche.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>1020</x>
       <y>30</y>
       <width>171</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>../../Desktop/ADMIN/BoutonTriC.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>1020</x>
       <y>100</y>
       <width>171</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>../../Desktop/ADMIN/BoutonTriD.png</pixmap>
     </property>
    </widget>
    <widget class="QLineEdit" name="Recher">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>110</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="Rechercher">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>50</y>
       <width>171</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="TriC_2">
     <property name="geometry">
      <rect>
       <x>1010</x>
       <y>30</y>
       <width>181</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="TriD_2">
     <property name="geometry">
      <rect>
       <x>1010</x>
       <y>100</y>
       <width>181</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="Afficher">
    <attribute name="title">
     <string>Afficher Abonnement</string>
    </attribute>
    <widget class="QTableWidget" name="AfficherAO">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>200</y>
       <width>1151</width>
       <height>321</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Source Sans Pro</family>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="autoScroll">
      <bool>false</bool>
     </property>
     <property name="autoScrollMargin">
      <number>20</number>
     </property>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>160</number>
     </attribute>
     <column>
      <property name="text">
       <string>ID Abonnement</string>
      </property>
      <property name="font">
       <font>
        <family>Source Sans Pro</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="background">
       <color alpha="0">
        <red>0</red>
        <green>0</green>
        <blue>255</blue>
       </color>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Type</string>
      </property>
      <property name="font">
       <font>
        <family>Source Sans Pro</family>
        <pointsize>14</pointsize>
       </font>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Prix</string>
      </property>
      <property name="font">
       <font>
        <family>Source Sans Pro</family>
        <pointsize>14</pointsize>
       </font>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Date Debut</string>
      </property>
      <property name="font">
       <font>
        <family>Source Sans Pro</family>
        <pointsize>14</pointsize>
       </font>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Date Fin</string>
      </property>
      <property name="font">
       <font>
        <family>Source Sans Pro</family>
        <pointsize>14</pointsize>
       </font>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Duree</string>
      </property>
      <property name="font">
       <font>
        <family>Source Sans Pro</family>
        <pointsize>14</pointsize>
       </font>
      </property>
     </column>
     <column>
      <property name="text">
       <string>ID Personne</string>
      </property>
      <property name="font">
       <font>
        <family>Source Sans Pro</family>
        <pointsize>14</pointsize>
       </font>
      </property>
     </column>
    </widget>
    <widget class="QComboBox" name="comboBox_3">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>60</y>
       <width>181</width>
       <height>41</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>ID_Abonnement</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Type</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Prix</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Duree</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>ID_Personne</string>
      </property>
     </item>
    </widget>
    <widget class="QComboBox" name="comboBox_4">
     <property name="geometry">
      <rect>
       <x>880</x>
       <y>60</y>
       <width>181</width>
       <height>41</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>ID_Abonnement</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Date_Fin</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Date_Debut</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Type</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Prix</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Duree</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>ID_Personne</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="IDC">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>70</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>ID Cours</string>
     </property>
    </widget>
    <widget class="QLabel" name="IDC_2">
     <property name="geometry">
      <rect>
       <x>780</x>
       <y>70</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>ID Cours</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>1070</x>
       <y>30</y>
       <width>171</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>../../Desktop/ADMIN/BoutonTriC.png</pixmap>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>1070</x>
       <y>90</y>
       <width>171</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>../../Desktop/ADMIN/BoutonTriD.png</pixmap>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>60</y>
       <width>160</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>../../Desktop/ADMIN/Recherche.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="Recher_2">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>120</y>
       <width>180</width>
       <height>40</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="Recherche">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>50</y>
       <width>161</width>
       <height>71</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="TriC">
     <property name="geometry">
      <rect>
       <x>1070</x>
       <y>30</y>
       <width>171</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="TriD">
     <property name="geometry">
      <rect>
       <x>1070</x>
       <y>90</y>
       <width>171</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
